package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "fingerprint-pro-server-api-go-sdk",
+  "name": "fingerprint-server-api-go-sdk",
```

Comment:
I couldn't find any place where this is relevant in the Go SDK (it does not change the SDK's package name), so maybe this field is relevant in "dx-team-toolkit" repo. @TheUnderScorer could you give your opinion on this change?

Reply:
you can change it or revert it, final decision is up to you guys @erayaydin @TheUnderScorer

Reply:
I'm fine with changing it, but in that case let's do the same for other server-side SDKs. If it's too much work, let's revert it and stick with old name.